mreg/models.py

```diff
@@ -786,3 +787,25 @@ class History(models.Model):
 
     def __str__(self):
         return f'{self.name}, {self.model}, {self.action}, {self.timestamp}'
+
+
+class BACnetID(models.Model):
+    id = models.IntegerField(primary_key=True, validators=[validate_BACnetID])
+    host = models.OneToOneField(Host, on_delete=models.CASCADE, related_name='bacnetid')
```

Comment:
Why a full model for just a field? Because few hosts will have it or what?

Reply:
I also considered just adding a field to Host. Noticed that Loc and Hinfo are one-to-one-fields with their own model, and ended up choosing a similar pattern for this solution. Not sure what would be best